tests/mappings/modules/rates/pot/handleFile.test.ts

```diff
@@ -0,0 +1,71 @@
+import { Bytes, Address, BigInt, BigDecimal } from '@graphprotocol/graph-ts'
+import { units, bytes } from '@protofire/subgraph-toolkit'
+import { test, clearStore, assert, log, describe, beforeEach } from 'matchstick-as'
+import { LogNote } from '../../../../../generated/Pot/Pot'
+import { handleFile } from '../../../../../src/mappings/modules/rates/pot'
+import { mockDebt } from '../../../../helpers/mockedFunctions'
+import { tests } from '../../../../../src/mappings/modules/tests'
+
+function strRadToBytes(value: string): Bytes {
+  return Bytes.fromUint8Array(Bytes.fromBigInt(BigInt.fromString(value)).reverse())
+}
+
+function createEventDSR(what: string, data: string): LogNote {
+  let sig = tests.helpers.params.getBytes('sig', Bytes.fromHexString('0x29ae8114'))
+  let usr = tests.helpers.params.getBytes('arg1', Bytes.fromUTF8(''))
+  let dataBytes = strRadToBytes(data)
+  let arg1 = tests.helpers.params.getBytes('arg2', Bytes.fromUTF8(what))
+  let arg2 = tests.helpers.params.getBytes('arg3', dataBytes)
+
+  let event = changetype<LogNote>(tests.helpers.events.getNewEvent([sig, usr, arg1, arg2]))
+
+  return event
+}
+
+function createEventVow(what: string, data: string): LogNote {
```

Comment:
I looks like the only difference between the 2 event creator functions is the signature. Please pass the signature as a parameter.